lib/dialects/oracle/index.js

```diff
@@ -80,10 +79,10 @@ Object.assign(Client_Oracle.prototype, {
   // Get a raw connection, called by the `pool` whenever a new
   // connection needs to be added to the pool.
   acquireRawConnection() {
-    return new Bluebird((resolver, rejecter) => {
+    return new Promise((resolver, rejecter) => {
       this.driver.connect(this.connectionSettings, (err, connection) => {
         if (err) return rejecter(err);
-        Bluebird.promisifyAll(connection);
+        connection.executeAsync = promisify(connection.execute);
```

Comment:
This seems like it is probably reasonable.  I'll have a better idea once I finish reviewing `dialects/oracledb`

Reply:
Confirmed -- this appears to be fine.  But, we might want to refactor this in a future commit.  (In particular: it's injecting data into a 3rd-party component.  This could become a problem if the component's interface changes in the future)